plugins/exporter.koplugin/target/my_clippings.lua

```diff
@@ -0,0 +1,51 @@
+local util = require("ffi/util")
+local T = util.template
+local _ = require("gettext")
+
+-- text exporter
+local TextExporter = require("base"):new {
```

Comment:
It won't affect the behaviour but I would like to rename the class to `ClippingsExporter` or similar. Same for the methods